<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceResetExcisionPage</class>
 <widget class="QWidget" name="MaintainceResetExcisionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_CT1_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TC1_2">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Mp1">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M1_1">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M1_3">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M1_2">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M2_1">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M2_2">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_M2_3">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Mp2">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TC2_1">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TC2_2">
   <property name="geometry">
    <rect>
     <x>885</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Excision_TC1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Reset_TC1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Reset_M1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Excision_M1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Excision_M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Reset_M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Excision_TC2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SIV_Reset_TC2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>SIV复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_F_Excision_Mp1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Reset_Mp1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Ele_Excision_Mp1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU电制动
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_F_Excision_M1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_F_Excision_M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_F_Excision_Mp2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Reset_M1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Ele_Excision_M1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU电制动
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Reset_M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Ele_Excision_M2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU电制动
切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Reset_Mp2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU逆变器
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DCU_Ele_Excision_Mp2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(187,188,247);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>DCU电制动
切除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
